### DISCUSSION TITLE
Page flickers when clicking a link

### DISCUSSION THREAD

**Discussion by heyjoe1984** [livewire/livewire#4396] [7. Bugs (archived)]
I'm noticing some times (well more than half of the time) the page UI seems to flicker in between requests.

See this video: https://imgur.com/a/laY8j0c

It looks like Livewire is prefetching the next page and it causes an error in the JS while its fetching? Is this correct?

When the livewire.js goes red in the dev tools, it says "(cancelled)". Is there any way for me to fix this?

Edit: I just noticed chrome is reporting an issue, not sure if this is the cause:

> A page or script is accessing at least one of navigator.userAgent, navigator.appVersion, and navigator.platform. Starting in Chrome 101, the amount of information available in the User Agent string will be reduced.
To fix this issue, replace the usage of navigator.userAgent, navigator.appVersion, and navigator.platform with feature detection, progressive enhancement, or migrate to navigator.userAgentData.
Note that for performance reasons, only the first access to one of the properties is shown.
> AFFECTED RESOURCES
> 1 source
> livewire.js

This doesn't seem to happen in Safari.

**Comment by joshhanley** [comment]
@heyjoe1984 for some reason your video link won't load for me so I can't review it.

Also could you share your code which reproduces the issue with as little code as possible so we can investigate it?

Thanks!

**Reply by heyjoe1984** [reply]
@joshhanley try the video now, Imgur was down last night

**Comment by joshhanley** [comment]
@heyjoe1984 just had a look at your video and yeah that is strange.

Any chance you could create a small example repo which reproduces the issue with as little code as possible so we can investigate it?

Thanks!
